lib/src/validation-server/server.ts

```diff
@@ -0,0 +1,23 @@
+import express from "express";
+import { Contract } from "../neu/definitions";
+
+export function runValidationServer(
+  port: number,
+  contract: Contract,
+  logger: Logger
+) {
+  const app = express();
+
+  app.get("/health", (req, res) => {
+    res.status(200).end();
+  });
+
+  return {
+    app,
+    defer: () => new Promise(resolve => app.listen(port, resolve))
+  };
+}
+
+export interface Logger {
```

Comment:
Is this useful to other parts of the Spot codebase? Should we consider moving it to a more central spot?

Reply:
Yes, I thought about that too, I'm happy to move it